resources/lang/en.json

```diff
@@ -682,11 +688,12 @@
   "success": "success",
   "Successfully imported": "Successfully imported",
   "Successfully saved": "Successfully saved",
-  "Summary screen that will be displayed when process finish with this End event.": "Summary screen that will be displayed when process finish with this End event.",
```

Comment:
This was replaced with "Select Display-type Screen to show the summary of this Request when it completes" ✅  and "Select Screen to display this Task" ❌